README.md

```diff
@@ -165,13 +165,14 @@ If you're using sbt, add the following to your build:
 ```sbt
 libraryDependencies ++= Seq(
   "eu.timepit" %% "refined"            % "0.6.0",
-  "eu.timepit" %% "refined-scalaz"     % "0.6.0",         // optional
-  "eu.timepit" %% "refined-scodec"     % "0.6.0",         // optional
-  "eu.timepit" %% "refined-scalacheck" % "0.6.0" % "test" // optional
+  "eu.timepit" %% "refined-scalaz"     % "0.6.0",          // optional
+  "eu.timepit" %% "refined-scodec"     % "0.6.0",          // optional
+  "eu.timepit" %% "refined-scalacheck" % "0.6.0" % "test", // optional
+  "eu.timepit" %% "refined-pureconfig" % "0.6.0"           // optional
 )
 ```
 
-For Scala.js just replace `%%` with `%%%` above.
+For Scala.js just replace `%%` with `%%%` above (except for `refined-pureconfig` which is not available for Scala.js).
```

Comment:
Issue for pureconfig scala.js support: https://github.com/melrief/pureconfig/issues/46